testing/simulator/src/checks.rs

```diff
@@ -172,15 +173,20 @@ pub async fn verify_fork_version<E: EthSpec>(
 /// have full aggregates.
 pub async fn verify_full_sync_aggregates_up_to<E: EthSpec>(
     network: LocalNetwork<E>,
+    db: Arc<HotColdDB<E, LevelDB<E>, LevelDB<E>>>,
     sync_committee_start_slot: Slot,
     upto_slot: Slot,
     slot_duration: Duration,
 ) -> Result<(), String> {
     slot_delay(upto_slot, slot_duration).await;
+    let anchor_info = db.get_anchor_info();
+    let state_upper_limit = anchor_info.state_upper_limit;
+    let effective_start_slot = std::cmp::max(sync_committee_start_slot, state_upper_limit);
```

Comment:
I think we can probably just revert all the changes in this file. These tests run with historic states enabled, so it doesn't make any difference.

If you would like to add a test, have a look at `beacon_node/beacon_chain/tests/store_tests.rs`. We could add a test that checkpoint syncs a node from some state, and then advances the chain and queries a historic sync committee which starts prior to the checkpoint. We would likely want to use the `MinimalEthSpec` so that sync committees aren't too long.

Reply:
have removed it , can you run the pipeline once, in my recent local setup , i am unable to see tc passing/failing